autoapi/mappers/base.py

```diff
@@ -176,14 +210,19 @@ def create_class(self, obj):
     def output_rst(self, root, source_suffix):
         for id, obj in self.objects.items():
 
-            if not obj:
+            if not obj or not obj.top_level_object:
                 continue
 
             rst = obj.render()
             if not rst:
                 continue
 
-            detail_dir = os.path.join(root, *id.split('.'))
+            try:
+                filename = id.split('(')[0]
+            except IndexError:
+                filename = id
+            filename = filename.replace('#', '-')
```

Comment:
Yea, it's a .Netism, just didn't want to have to override that whole function in the .Net code. Probably means we need a smaller level of abstraction here, like having a "filename" on the obj.